tests/test_bfh.py

```diff
@@ -349,6 +358,38 @@ class Mymap(Mapping):
 
         self.assertEqual({"cool": 1}, transformed.serialize())
 
+    def test_empty_fields_serialized_with_defaults(self):
+        class FirstSchema(Schema):
+            wow = IntegerField(required=True)
+            umm = IntegerField(required=False, default=2)
```

Comment:
related to my comment on permitting a callable as default, i'd like to see a broader variety of types tested as defaults